codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentBuilderGenerator.kt

```diff
@@ -150,6 +162,74 @@ class FluentBuilderGenerator(
                 }
                 """,
                 *scope,
+                "epilogue" to
+                    writable {
```

Comment:
this could probably be cleaned up into well defined functions that make it clearer it's handling event stream concerns (as opposed to an inline `writable { ... }`

Reply:
Agreed, moved to a helper in fe93c5c.